articles/tokens/refresh-token.md

```diff
@@ -1,152 +1,173 @@
+- `client_secret`: Your application's Client Secret.
+- `token`: The refresh token you want to revoke.
 
-To revoke a __refresh token__, call the [Delete a device credential](/api/management/v2#!/Device_Credentials/delete_device_credentials_by_id) endpoint with an access token containing `delete:device_credentials` scope and the value of `id` obtained above:
+The client should match the one the refresh token was issued for.
 
-```text
-DELETE https://${account.namespace}/api/v2/device-credentials/{id}
+If the request is valid, the refresh token is revoked and the response is `HTTP 200`, with an empty response body. Otherwise, the response body contains the error code and description.
 
+```json
 {
-  "Authorization":   "Bearer {your_access_token}"
+  "error": "invalid_request|invalid_client",
+  "error_description": "Description of the error"
 }
-
 ```
 
-The response will be a **204**: The credential no longer exists.
+The possible responses are:
+
+| HTTP Status | Description |
+| --- | --- |
+| 200 | The refresh token is revoked or does not exist. The response body is empty. |
+| 400 | The required parameters were not sent in the request or the refresh token was not issued to the client making the revocation request (`"error": "invalid_request"`). |
+| 401 | The request is not authorized (`"error": "invalid_client"`). Check that the client credentials (`client_id` and `client_secret`) are present in the request and hold valid values. |
 
-### Revoke a Refresh Token in the Dashboard
+### Revoke a refresh token using the dashboard
 
-To see if a user has existing devices with associated refresh tokens, go to the [Users section](${manage_url}/#/users) of the dashboard. Click the name of the user to view their **Details** page.
+To revoke a refresh token, go to the [Users section](${manage_url}/#/users) of the [dashboard](${manage_url}). Click the name of the user to view their *Details* page.
 
-Select the **Devices** tab. This page lists all device names and the number of refresh tokens associated with each. To revoke a refresh token, click the **X** to the right of the device name.
+Select the *Authorized Applications* tab. This page lists all the clients to which the user has authorized access. Revoking an authorized application revokes also its associated refresh tokens.
 
-![](/media/articles/tokens/dashboard-revoke-refresh-token.png)
+To revoke a refresh token, click **Revoke**.
 
-Click **UNLINK** to confirm.
+![Revoke a refresh token using the dashboard](/media/articles/tokens/dashboard-revoke-refresh-token.png)
 
 ## SDK Support
 
-The [Lock](/libraries/lock), [auth0.js](/libraries/auth0js), and [auth0-angular.js](https://github.com/auth0/auth0-angular) libraries include support to obtain and use refresh tokens.
+The [Lock](/libraries/lock), [auth0.js](/libraries/auth0js), and [auth0-angular.js](https://github.com/auth0/auth0-angular) libraries include support methods to obtain and use refresh tokens.
```

Comment:
Remove this p. The mantra is "no refresh tokens should be placed in the browser". If any of our front end SDK support this we need to remove it.